core/theme_manager.js

```diff
@@ -113,7 +113,9 @@ ThemeManager.prototype.setTheme = function(theme) {
     }
   }
 
-  Blockly.hideChaff();
+  for (let i = 0, workspace; (workspace = this.subscribedWorkspaces_[i]); i++) {
```

Comment:
(Obligatory remark that the style guide says to prefer `for...of`.)

Reply:
Updated accordingly.